src/amo/components/AddonHead/index.js

```diff
@@ -0,0 +1,225 @@
+            i18n.gettext(`%(addonName)s – Get this Language Pack for 🦊 Android
+              (%(locale)s)`),
+            i18nValues,
+          );
+        case ADDON_TYPE_STATIC_THEME:
+        case ADDON_TYPE_THEME:
+          return i18n.sprintf(
+            // translators: please keep the fox emoji next to "Android".
+            i18n.gettext(
+              `%(addonName)s – Get this Theme for 🦊 Android (%(locale)s)`,
+            ),
+            i18nValues,
+          );
+        case ADDON_TYPE_OPENSEARCH:
+          return i18n.sprintf(
+            // translators: please keep the fox emoji next to "Android".
+            i18n.gettext(`%(addonName)s – Get this Search Tool for 🦊 Android
+              (%(locale)s)`),
+            i18nValues,
+          );
+        default:
+          return i18n.sprintf(
+            // translators: please keep the fox emoji next to "Android".
+            i18n.gettext(`%(addonName)s – Get this Add-on for 🦊 Android
+              (%(locale)s)`),
+            i18nValues,
+          );
+      }
+    }
+
+    switch (addon.type) {
+      case ADDON_TYPE_DICT:
+        return i18n.sprintf(
+          // translators: please keep the fox emoji next to "Firefox".
+          i18n.gettext(`%(addonName)s – Get this Dictionary for 🦊 Firefox
+            (%(locale)s)`),
+          i18nValues,
+        );
+      case ADDON_TYPE_EXTENSION:
+        return i18n.sprintf(
+          // translators: please keep the fox emoji next to "Firefox".
+          i18n.gettext(`%(addonName)s – Get this Extension for 🦊 Firefox
+            (%(locale)s)`),
+          i18nValues,
+        );
+      case ADDON_TYPE_LANG:
+        return i18n.sprintf(
+          // translators: please keep the fox emoji next to "Firefox".
+          i18n.gettext(`%(addonName)s – Get this Language Pack for 🦊 Firefox
+            (%(locale)s)`),
+          i18nValues,
+        );
+      case ADDON_TYPE_STATIC_THEME:
+      case ADDON_TYPE_THEME:
+        return i18n.sprintf(
+          // translators: please keep the fox emoji next to "Firefox".
+          i18n.gettext(`%(addonName)s – Get this Theme for 🦊 Firefox
+            (%(locale)s)`),
+          i18nValues,
+        );
+      case ADDON_TYPE_OPENSEARCH:
+        return i18n.sprintf(
+          // translators: please keep the fox emoji next to "Firefox".
+          i18n.gettext(`%(addonName)s – Get this Search Tool for 🦊 Firefox
+            (%(locale)s)`),
+          i18nValues,
+        );
+      default:
+        return i18n.sprintf(
+          // translators: please keep the fox emoji next to "Firefox".
+          i18n.gettext(`%(addonName)s – Get this Add-on for 🦊 Firefox
+            (%(locale)s)`),
+          i18nValues,
+        );
+    }
+  }
+
+  getPageDescription() {
+    const { addon, i18n } = this.props;
+
+    invariant(addon, 'addon is required');
+
+    return i18n.sprintf(
+      i18n.gettext('Download %(addonName)s for Firefox. %(summary)s'),
```

Comment:
I don't think so here, because in both cases it is an add-on for Firefox, right? I mean, downloading something for Android would mean an Android app to me, not a FF app.

This is super confusing, but if we get a better title like "Firefox for Android" in https://github.com/mozilla/addons-frontend/issues/6648, I'll make the change here too